Tests/UIKitTests/UICollectionViewExtensionsTests.swift

```diff
@@ -75,6 +92,41 @@ final class UICollectionViewExtensionsTests: XCTestCase {
     }
     #endif
 
+    func testSafeScrollToIndexPath() {
+        let validIndexPathTop = IndexPath(row: 0, section: 0)
+
+        flowLayoutCollectionView.contentOffset = CGPoint(x: 0, y: 30)
+        XCTAssertNotEqual(flowLayoutCollectionView.contentOffset, .zero)
+
+        flowLayoutCollectionView.safeScrollToItem(at: validIndexPathTop, at: .top, animated: false)
+        XCTAssertEqual(flowLayoutCollectionView.contentOffset, .zero)
+
+        let validIndexPathBottom = IndexPath(row: 4, section: 0)
+
+        let bottomOffset = CGPoint(x: 0, y: flowLayoutCollectionView.collectionViewLayout.collectionViewContentSize.height - flowLayoutCollectionView.bounds.size.height)
+
+        flowLayoutCollectionView.contentOffset = .init(x: 0, y: 30)
```

Comment:
It's inconsistent to use `.init` here.
```suggestion
        flowLayoutCollectionView.contentOffset = CGPoint(x: 0, y: 30)
```